<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lista Jednokierunkowa</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>391</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="lbCircle">
           <property name="text">
            <string>Dodaj okrąg:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="lbX">
               <property name="text">
                <string>x:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leX">
               <property name="text">
                <string/>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="lbY">
               <property name="text">
                <string>y:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leY">
               <property name="text">
                <string/>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="lbRadius">
               <property name="text">
                <string>radius:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leRadius">
               <property name="text">
                <string/>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="lbRadius_2">
           <property name="text">
            <string>Przedział promienia:</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="lbY_2">
               <property name="text">
                <string>od:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leFrom">
               <property name="text">
                <string/>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="lbY_3">
               <property name="text">
                <string>do:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leTo">
               <property name="text">
                <string/>
               </property>
               <property name="clearButtonEnabled">
                <bool>false</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="pbShow">
           <property name="text">
            <string>Pokaż</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbSave">
           <property name="text">
            <string>Zapisz</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbRead">
           <property name="text">
            <string>Wczytaj</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbClose">
           <property name="text">
            <string>Zamknij program</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QPlainTextEdit" name="pteList">
         <property name="maximumSize">
          <size>
           <width>241</width>
           <height>171</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="pbAddAtTheFront">
           <property name="text">
            <string>Dodaj na początek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbAddAtTheEnd">
           <property name="text">
            <string>Dodaj na koniec</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchBy">
           <property name="text">
            <string>Wyszukaj okregi o promieniu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SortByOrigin">
           <property name="text">
            <string>Sortuj od początku układu</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbDelete">
           <property name="text">
            <string>Usuń </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbClear">
           <property name="text">
            <string>Wyczyść</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>490</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionZamknij"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionZamknij">
   <property name="icon">
    <iconset>
     <normaloff>../../Downloads/remove.png</normaloff>../../Downloads/remove.png</iconset>
   </property>
   <property name="text">
    <string>Zamknij</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
